.travis.yml

```diff
@@ -6,3 +6,9 @@ go:
   - '1.12'
   - '1.13'
   - 'tip'
+
+script:
+  - go test -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
The script will only check the top-level package.

Reply:
Forgot to add the `./...` at the end. Added.